contrib/platform/src/com/sun/jna/platform/win32/Psapi.java

```diff
@@ -266,6 +266,46 @@
      */
     boolean GetPerformanceInfo(PERFORMANCE_INFORMATION pPerformanceInformation, int cb);
 
+    /**
+     * It is a good idea to use a large array, because it is hard to predict
+     * how many processes there will be at the time you call EnumProcesses.
+     * <br>
+     * To determine how many processes were enumerated, divide the
+     * pBytesReturned value by sizeof(DWORD). There is no indication given
+     * when the buffer is too small to store all process identifiers.
+     * Therefore, if pBytesReturned equals cb, consider retrying the call
+     * with a larger array.
+     * To obtain process handles for the processes whose identifiers you
+     * have just obtained, call the OpenProcess function.<br>
+     * Starting with Windows 7 and Windows Server 2008 R2, Psapi.h
+     * establishes version numbers for the PSAPI functions. The PSAPI
```

Comment:
I'm not sure we need these details as they are hidden from the user, we always just call `EnumProcesses`.

Reply:
The sole reason why I left it there was that it was documented on MSDN. As this part is not relevant for the JNA layer, I removed it in the updated commit.